docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
```

Comment:
What about simplifying the last part?

... your Virtual Organization entitles you to, you can use VOMS authentication.